newdle/client/src/components/MyNewdles.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {Container, Icon, Label} from 'semantic-ui-react';
+import {serializeDate, toMoment} from '../util/date';
+import styles from './MyNewdles.module.scss';
+
+export default function MyNewdles() {
+  const dummyData = [
+    {
+      id: 1,
+      title: 'Indico weekly meeting',
+      participantsCount: 5,
+      participantsTotal: 7,
+    },
+    {
+      id: 2,
+      title: '2.2 Release Reflection Workshop',
+      participantsCount: 7,
+      participantsTotal: 7,
+      duration: 60,
+      finalDt: '2019-07-1 10:30:00',
+      timezone: 'Europe/Zurich',
+    },
+  ];
+  return (
+    <Container text className={styles.newdles}>
+      {dummyData.map(newdle => (
+        <div
+          className={styles.newdle}
+          key={newdle.id}
+          onClick={() => console.log('go to newdle summary')}
```

Comment:
`const router = useRouter();` and then `router.history.push(`/newdle/${newdle.code}/summary`);`